portal-kernel/test/unit/com/liferay/portal/kernel/util/SystemPropertiesTest.java

```diff
@@ -26,6 +26,7 @@
 
 import java.util.Map;
 import java.util.Properties;
+import java.util.Set;
```

Comment:
@W2jx When you say "Replace", you should always tell people you replace what with what, thus the "Replace unit test" is confusing. In your case, the reason you modify unit test is because the class you tested is changed, so here you can just say "Fix the unit test".

Please update the message.

Thanks.

Reply:
I will fix it.